test/alternator/test_table.py

```diff
@@ -442,31 +442,6 @@ def test_update_table_non_existent(dynamodb, test_table):
     with pytest.raises(ClientError, match='ResourceNotFoundException'):
         client.update_table(TableName=random_string(20), BillingMode='PAY_PER_REQUEST')
 
-# Consistent schema change feature is optionally enabled and
-# some tests are expected to fail on Scylla without this
-# option enabled, and pass with it enabled (and also pass on Cassandra).
-# These tests should use the "fails_without_consistent_cluster_management"
-# fixture. When consistent mode becomes the default, this fixture can be removed.
-@pytest.fixture(scope="session")
-def check_pre_consistent_cluster_management(dynamodb):
-    from util import is_aws
-    # If not running on Scylla, return false.
-    if is_aws(dynamodb):
-        return false
-    # In Scylla, we check Raft mode by inspecting the configuration via a
-    # system table (which is also visible in Alternator)
-    config_table = dynamodb.Table('.scylla.alternator.system.config')
-    consistent = config_table.query(
-            KeyConditionExpression='#key=:val',
-            ExpressionAttributeNames={'#key': 'name'},
-            ExpressionAttributeValues={':val': 'consistent_cluster_management'}
-        )['Items']
-    return len(consistent) == 0 or consistent[0]['value'] == 'false'
-@pytest.fixture(scope="function")
-def fails_without_consistent_cluster_management(request, check_pre_consistent_cluster_management):
-    if check_pre_consistent_cluster_management:
-        request.node.add_marker(pytest.mark.xfail(reason='Test expected to fail without consistent cluster management feature on'))
```

Comment:
I'm not sure we should remove this test-skipping fixture (and its uses). Keeping it for a while longer will allow us to run modern Alternator tests on *older* Scylla. Which is a worthy goal. There is no rush to remove this code now.

Reply:
From `test/alternator/README.md`:

`run` automatically picks the most-recently compiled version of Scylla in
`build/*/scylla` - but this choice of Scylla executable can be overridden with
the `SCYLLA` environment variable.

I didn't know we can do that. I'll bring these fixtures back.

Reply:
I also removed the support for the `--raft` option from `test/alternator/run` and `test/cql-pytest/run`. Is it fine? I think this option has already been outdated since `consistent_cluster_management=true` became the default. Tests use raft even without this option unless we specify `consistent_cluster_management=false` directly.

Also, I don't think we want to keep this option for even older Scylla versions with `consistent_cluster_management=false` as the default. Having a flag that normally has no effect seems to be very confusing.

Reply:
Yes, I think what you did now is good. The test themselves can still run against ancient Scylla versions if you run it yourself - but the run script only knows how to run a modern version of Scylla - I think that's fine.

If we had any possiblity of turning off raft, we could have left an "--no-raft" option to the run script, but if I understand correctly master no longer allows turning raft off, so we don't even need that. 

So I reviewed your patches and everything seems ok as far as I concerned. But the bigger Raft experts should review it for actual raft content (and documentation changes, etc.) so I didn't leave a review decision. I'll just resolve this discussion. Thanks.